collector/zfs_linux.go

```diff
@@ -79,7 +157,7 @@ func (c *zfsCollector) parseArcstatsProcfsFile(reader io.Reader, handler func(zf
 
 	}
 	if !parseLine {
-		return errors.New("did not parse a single arcstat metric")
+		return fmt.Errorf("did not parse a single %s metric", fmt_ext)
```

Comment:
@SuperQ Actually, this is intended to be an error message, so I went ahead and updated to %q. Thanks!